pegasus/helpers/section_api_helpers.rb

```diff
@@ -159,6 +165,11 @@ def self.valid_gender?(gender)
     VALID_GENDERS.include?(gender)
   end
 
+  VALID_PROVIDERS = %w(google_oauth2)
```

Comment:
The name of this is a bit misleading, as lots of other providers are "valid". Perhaps `pegasus_supported_provider?`?